### ISSUE TITLE
Add more parameters to MLP trainer method

### ISSUE THREAD

**Issue by FlorentinD** [neo4j/graph-data-science#242]
**Is your feature request related to a problem? Please describe.**
In the machine learning pipelines, you can use different trainer methods when adding a model candidate, such as RandomForest or MLP.
For MLP, we could expose more parameters to configure.

Current exposed parameters are listed at https://neo4j.com/docs/graph-data-science/current/machine-learning/training-methods/mlp/.

**Describe the solution you would like**

We could also expose other optional parameters such as:

- [ ] Different activiation functions (in addition to Softmax)
- [ ] Dropouts
- [ ] Optimizer

These are just examples, you could also come up with others.

**Additional context**

Good starting points are `MLPClassifierTrainConfig` and `MLPClassifier`

If you want to work on this issue please drop a comment :)
